components/FlatCard/__stories__/FlatCard.stories.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import { withKnobs, text } from '@storybook/addon-knobs';
+
+import { s3 } from 'common/constants/urls';
+
+import FlatCard from '../FlatCard';
+
+storiesOf('Common/FlatCard', module)
+  .addDecorator(withKnobs)
+  .add(
+    'default',
```

Comment:
I think it's best if we just do one story (with knobs) 👍 - the dev can find out how the knobs change the component